formats/json/commonTest/src/kotlinx/serialization/json/JsonParserFailureModesTest.kt

```diff
@@ -54,6 +54,14 @@ class JsonParserFailureModesTest : JsonTestBase() {
                     it
                 )
             }
+            // 9223372036854775807 is Long.MAX_VALUE
+            assertFailsWith<JsonDecodingException> {
+                default.decodeFromString(
+                    Holder.serializer(),
+                    """{"id":922337203685477580700}""",
```

Comment:
It's not very convenient to spot the problem here. Maybe replace this with `Long.MAX_VALUE.toString() + "00"`?

Reply:
Sure